sql/database_test.go

```diff
@@ -659,3 +660,32 @@ func TestConnection(t *testing.T) {
 		return errors.New("error")
 	}))
 }
+
+func TestConnection_Idle(t *testing.T) {
+	dbName := t.Name()
+	numConns := func() int {
+		return int(testutil.ToFloat64(PoolUsage.WithLabelValues(dbName)))
+	}
```

Comment:
This can be inlined, if the type check of `require.Equal` is the reason for this function you can also use `require.EqualValues` which accepts `float64(1) == int(1)`

Reply:
This is done b/c `ToFloat64` will always return integer values as `float64` here, but if we treat them as such we get a linter error:
```
./bin/golangci-lint run --config .golangci.yml
sql/database_test.go:672:6: float-compare: use require.InEpsilon (or InDelta) (testifylint)
					require.EqualValues(t, 1, testutil.ToFloat64(PoolUsage.WithLabelValues(dbName)))
					^
sql/database_test.go:677:5: float-compare: use require.InEpsilon (or InDelta) (testifylint)
				require.EqualValues(t, 1, testutil.ToFloat64(PoolUsage.WithLabelValues(dbName)))
				^
```
There's no sense in using `require.InEpsilon` / `require.InDelta` here, and silencing the linter warning explicitly is not much better than `numConns` func, which also makes the intent of reading the metric more clear.